ArduPlane/GCS_Mavlink.cpp

```diff
@@ -699,6 +699,8 @@ bool GCS_MAVLINK_Plane::try_send_message(enum ap_message id)
     case MSG_AOA_SSA:
         CHECK_PAYLOAD_SIZE(AOA_SSA);
         plane.send_aoa_ssa(chan);
+    case MSG_LANDING:
+        plane.landing.send_landing_message(chan);
```

Comment:
yeah, I see what you mean about the packet size. However, the least you could do is skip it entirely if you're not in the LAND flight stage. I know it fixes itself on the backend but from the top level here it looks like it logs and its not nice to hide it at this level